internal/engine/experiment/riseupvpn/riseupvpn.go

```diff
@@ -244,20 +253,21 @@ func (m Measurer) Run(ctx context.Context, sess model.ExperimentSession,
 	overallCount := 1 + len(inputs) + len(openvpnEndpoints) + len(obfs4Endpoints)
 
 	// measure openvpn in parallel
-	multi = urlgetter.Multi{Begin: measurement.MeasurementStartTimeSaved, Getter: m.Getter, Session: sess}
-	for entry := range multi.CollectOverall(ctx, openvpnEndpoints, 1+len(inputs), overallCount, "riseupvpn", callbacks) {
+	for entry := range multi.CollectOverall(
+		ctx, openvpnEndpoints, 1+len(inputs), overallCount, "riseupvpn", callbacks) {
 		testkeys.AddGatewayConnectTestKeys(entry, "openvpn")
 	}
 
 	// measure obfs4 in parallel
-	multi = urlgetter.Multi{Begin: measurement.MeasurementStartTimeSaved, Getter: m.Getter, Session: sess}
-	for entry := range multi.CollectOverall(ctx, obfs4Endpoints, 1+len(inputs)+len(openvpnEndpoints), overallCount, "riseupvpn", callbacks) {
+	// TODO(bassosimone): when urlgetter is able to do obfs4 handshakes, here
```

Comment:
nice, is it on a project-wide todo in order to improve the tests for tor as well?

Reply:
To extend obfs4 capabilities to every test that needs it. The `tor` test already uses `obfs4` via a legacy codepath that I need to cleanup and make available to every test.